docs/books/admin_guide/07-file-systems.md

```diff
@@ -73,14 +95,14 @@ sudo fdisk -l /dev/sdc2
 
 ### `parted` command
 
-The `parted` (_partition editor_) command is able to partition a disk.
+The `parted` (_partition editor_) command is able to partition a disk, it solves the shortcomings of `fdisk`, So we recommend that you use the `parted` command even more.
```

Comment:
shortcomings of `fdisk`. So we recommend that

Reply:
Ahh... one thing @SergeCroise : that would be a lower-case 's': "so we recommend that you use the `parted` command even more."